app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/model/CalendarItem.kt

```diff
@@ -124,22 +101,8 @@ data class CalendarItem(
         val textColor = if (isCanceled) color else Color.WHITE
         val borderWidth = if (isCanceled) 2 else 0
 
-        val style = WeekViewEvent.Style.Builder()
-            .setBackgroundColor(backgroundColor)
-            .setTextColor(textColor)
-            .setTextStrikeThrough(isCanceled)
-            .setBorderWidth(borderWidth)
-            .setBorderColor(color)
-            .build()
-
-        return WeekViewEvent.Builder<CalendarItem>(this)
-            .setId(nr.toLong())
-            .setTitle(title)
-            .setStartTime(eventStart.toGregorianCalendar())
-            .setEndTime(eventEnd.toGregorianCalendar())
-            .setLocation(location)
-            .setStyle(style)
-            .setAllDay(false)
-            .build()
+        val style = WeekViewEvent.Style.Builder().setBackgroundColor(backgroundColor).setTextColor(textColor).setTextStrikeThrough(isCanceled).setBorderWidth(borderWidth).setBorderColor(color).build()
+
+        return WeekViewEvent.Builder(this).setId(nr.toLong()).setTitle(title).setStartTime(eventStart.toGregorianCalendar()).setEndTime(eventEnd.toGregorianCalendar()).setLocation(location).setStyle(style).setAllDay(false).build()
```

Comment:
For builders I think separating this into multiple lines is a lot more readable.
Could you revert this change or is this mandated by another tool?